cmd/dlt/cluster/cmd.go

```diff
@@ -110,6 +110,27 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 }
 
+func checkDeletion(r *rosa.Runtime, cluster *cmv1.Cluster, clusterKey string) string {
```

Comment:
this is doing more than checking the deletion, this is actually handling the deletion. I would suggest renaming this fuction to be more descriptive of what it is doing, even just `handleClusterDelete` or something like that